documentation/docs/10-adapters.md

```diff
@@ -56,6 +57,67 @@ You can also use `adapter-static` to generate single-page apps (SPAs) by specify
 
 > You must ensure [`trailingSlash`](configuration#trailingslash) is set appropriately for your environment. If your host does not render `/a.html` upon receiving a request for `/a` then you will need to set `trailingSlash: 'always'` to create `/a/index.html` instead.
 
+#### GitHub Pages
```

Comment:
this section should really go in the adapter-static README (there's already a GitHub Pages section). maybe it should be an `npm` example instead of `pnpm`? many more people use it. or we could have something like

```
packages/adapter-static/examples/github-pages-npm/build.yaml
packages/adapter-static/examples/github-pages-pnpm/build.yaml
```